src/robusta/core/model/pods.py

```diff
@@ -95,6 +95,9 @@ def get_number_of_bytes_from_kubernetes_mem_spec(mem_spec: str) -> int:
             if len(mem_spec) > 1 and mem_spec[-1] in k8s_memory_factors:
                 return int(mem_spec[:-1]) * k8s_memory_factors[mem_spec[-1]]
 
+            if mem_spec.isdigit():
+                return int(mem_spec)
+
```

Comment:
If we're already fixing it, I would also add support for memory spec with the format of "129e6". (This format can be parsed to float)
Looks like this format is also allowed:
https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory